parl/remote/worker.py

```diff
@@ -76,7 +77,7 @@ def __init__(self, master_address, cpu_num=None, log_server_port=None):
         self._set_cpu_num(cpu_num)
         self.job_buffer = queue.Queue(maxsize=self.cpu_num)
         self._create_sockets()
-        self.check_version()
+        self.check_version_and_package()
```

Comment:
Please rename the function to `check_env_consistency`. Same modification suggestion at the client.py.

Reply:
fix